src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/TensorPrimitives.netcore.cs

```diff
@@ -2579,6 +2579,291 @@ public static Vector512<float> Invoke(Vector512<float> x, Vector512<float> y)
 #endif
         }
 
+        private readonly struct LogOperator : IUnaryOperator
+        {
+            // This code is based on `vrs4_logf` from amd/aocl-libm-ose
```

Comment:
We already have the right callouts in the thirdparty file, right?

Reply:
Yep! We have the general encompassing license notice here: https://github.com/dotnet/runtime/blob/main/THIRD-PARTY-NOTICES.TXT#L1139-L1164

And then the explicit callout like this in each function that's ported something, so users have something to refer back to/cross reference more easily.